notebooks/examples/python/Ship2ShipTransfers/01. Data Ingestion.py

```diff
@@ -0,0 +1,90 @@
+# Databricks notebook source
+# MAGIC %md ## Data Ingestion
+
+# COMMAND ----------
+
+from pyspark.sql.functions import *
+import mosaic as mos
+
+spark.conf.set("spark.databricks.labs.mosaic.geometry.api", "ESRI")
+spark.conf.set("spark.databricks.labs.mosaic.index.system", "H3")
+mos.enable_mosaic(spark, dbutils)
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC
+# MAGIC We begin with loading from a table. Here we use captured `AIS` data.
+# MAGIC
+# MAGIC - MMSI: unique 9-digit identification code of the ship - numeric
+# MAGIC - VesselName: name of the ship - string
+# MAGIC - CallSign: unique callsign of the ship - string
+# MAGIC - timestamp: timestamp of the AIS message - datetime
+# MAGIC - LAT: latitude of the ship (in degree: [-90 ; 90], negative value represents South, 91 indicates ‘not available’) - numeric
+# MAGIC - LON: longitude of the ship (in degree: [-180 ; 180], negative value represents West, 181 indicates ‘not available’) - numeric
+# MAGIC - SOG: speed over ground, in knots - numeric
+# MAGIC - Status: status of the ship - string
+
+# COMMAND ----------
+
+cargos = spark.read.table("esg.cargos")
```

Comment:
I am getting an exception here: `Table or view not found: esg.cargos;`

Reply:
I tried to change it to `"ship2ship.AIS"` but then got on the next cell `AnalysisException: Column 'longitude' does not exist.`

Reply:
Fixed 😊